metadata/UiPathDocumentOCR__2__metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "changeLog": "Release v2020.12.2",
+  "cpu": 0,
```

Comment:
Shouldn't be 1 or it doesn't affect anything?

Reply:
doesn't matter. But they specifically want GPU to be used.